### ISSUE TITLE
File title sometimes gets cut off

### ISSUE THREAD

**Issue by neurowave** [2d-inc/support#139]
![image 2018-11-28 at 9 32 29 am](https://user-images.githubusercontent.com/1817068/49170278-97580100-f2f0-11e8-8de3-bd81dc1c960d.png)

**Comment by neurowave** [comment]
This is happening on the file preview page… I think we just need to add a few extra pixels to the width calculation of the input field. This changed when we changed the fonts to be a bit thicker, but the invisible div we're using to calculate the div was also changed, so I'm not sure why it's off. Anyway, I suspect that just adding a pixel or two to the calculation will fix this.

![screen recording 2018-12-15 at 09 30 pm](https://user-images.githubusercontent.com/1817068/50050476-c91ee500-00b0-11e9-9181-4fe15d9ca796.gif)
